vercel.json

```diff
@@ -5,7 +5,7 @@
       "headers": [
         {
           "key": "Content-Security-Policy",
-          "value": "frame-ancestors https://app.safe.global"
+          "value": "frame-ancestors https://app.safe.global https://eth-dapp-browser-ne49wqncp-ledgerhq.vercel.app"
```

Comment:
I guess this should be the production value? Or at least at the production url additionally?

Reply:
Good point. I will add the production eth-dapp browser address here additionally. (Keeping the old one for testing purposes)